components/ui/ProgressFreemiumComponent.tsx

```diff
@@ -0,0 +1,39 @@
+import { Indicator, Root } from "@radix-ui/react-progress";
+import React, { useEffect, useState } from "react";
+
+export interface ProgressFreemiumComponentProps {
+  elapsedDays: number;
+  totalTrialDays: number;
+}
+const ProgressBarComponent: React.FC<ProgressFreemiumComponentProps> = ({
```

Comment:
Component name and prop type name don't match

Reply:
changed to match